test/multiple-element-interactions-test.js

```diff
@@ -42,53 +44,55 @@ describe("When multiple DOM elements are present", () => {
             });
         });
 
-        it("can read attributes from custom child element's prototypes", () => {
-            var DataSource = components.newElement();
-            DataSource.data = [1, 2, 3];
-            components.registerElement("data-source", { prototype: DataSource });
+        // Pending until we decide on a good solution
```

Comment:
Well, see, the test itself doesn't pass. I would try to make it pass, but only if I knew the code would work in the browser too.